task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -54,16 +54,24 @@ spec:
                    namespace: "default",
                    successes: 0,
                    note: "",
-                   failures: (.runs[].results // [])|map(.message.text) | unique
+                   failures: (.runs[].results | length)
                  }' sast_snyk_check_out.json || true)
+
+          # Log out the failing runs
+          if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+          then
+            echo "The sast-snyk-check test fails with the following runs:"
+            echo $(jq '.runs[].results' sast_snyk_check_out.json)
```

Comment:
Maybe we can use the `.runs[].results // []|map(.message.text)` jq filter here to display only the error messages.

Reply:
@dirgim Thanks for the comments. They are all updated.

Reply:
Thanks!